Questions/Q2.js

```diff
@@ -8,19 +8,15 @@
 	Please implement the function that names fibonacci.
  */
 
-	function fibonacci(num) {
-		// TODO implement the fibonacci logic
-
-
-
-
-
-
-
-
-	}
-
-
-	fibonacci(1) // output: 1,
-	fibonacci(4) // output: 3,
-	fibonacci(6) // output: 8
\ No newline at end of file
+function fibonacci(num) {
+  // TODO implement the fibonacci logic
```

Comment:
Error handling likes positive integer input.
The another or better solution, try recursive?